roofit/roofitcore/src/RooFactoryWSTool.cxx

```diff
@@ -2038,6 +2038,9 @@ std::string RooFactoryWSTool::SpecialsIFace::create(RooFactoryWSTool& ft, const
     
   } else if (cl == "lagrangianmorph") {
     // Perform syntax check. Warn about any meta parameters other than the ones needed
+    vector<string> func_args = {"fileName","observableName","couplings","folders"};
```

Comment:
```suggestion
    const std::array<std::string,4> funcArgs{{"fileName","observableName","couplings","folders"}};
```
I'd make this a const array, so it's on the stack and it's clear that it's not changed after. Also remember the RooFit code convention for variables is `camelCase`.
You should also `#include <array>` for this.